portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,83 @@
 <!DOCTYPE html>
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+<div id="niceback">
```

Comment:
I don't think the `<html>` tag should be enclosed by another tag. IIUC ("If I understand correctly"), "niceback" is used to set a background on the page. You can style the <html> attribute directly. See [css-tricks](https://css-tricks.com/perfect-full-page-background-image) for an example.